src/App.js

```diff
@@ -1,11 +1,26 @@
 import './App.css';
+import Navbar from './components/Navbar';
+import logo from "./logo.png";
 
 function App() {
   return (
     <div className="App">
-      <h1>
-        TrackMe.Ninja website
-      </h1>
+      <Navbar />
+      <div className='ninja-stub'>
+        <h1>
+          TrackMe Ninja
+        </h1>
+
+      </div>
+      <div className='app-tracker-add'>
+        <div className='add-tracker'>
+          <input type="text" className='add-tracker-input' />
+          <button className='add-tracker-button'>Track</button>
+        </div>
```

Comment:
Later, I would take this out into a separate component. This will eventually need local state and handlers or some sort of form management library.